### ISSUE TITLE
Monitor loss function

### ISSUE THREAD

**Issue by ShazAlvi** [opengeokube/kit4dl#49]
**Is your feature request related to a problem? Please describe.**
Defining validation or training loss as a monitored value, not just the metrics. 

**Describe the solution you'd like**
Check if the defined value in the training.checkpoint.monitor is "val_loss" or "train_loss", if yes don't bypass the check if it is defined as a metric. The rest of the code should be okay. 

**Describe alternatives you've considered**
The check in confmodel.py needs to be bypassed. Without passing this check, you can't define a metric to be monitored that is not in the list of models. 

**Additional context**
Not applicable.
